pkg/minikube/cni/flannel.yaml

```diff
@@ -168,9 +168,6 @@ spec:
           requests:
             cpu: "100m"
             memory: "50Mi"
-          limits:
```

Comment:
This change also seems unrelated.

Reply:
It is from upstream ([30b56a37](https://github.com/flannel-io/flannel/commit/30b56a3794737135bf69d983417dcaaf3f1798b8)), but should probably be mentioned separately.

Reply:
reverted.